.specify/scripts/bash/check-prerequisites.sh

```
@@ -0,0 +1,170 @@
+# Include tasks.md if requested and it exists
+if $INCLUDE_TASKS && [[ -f "$TASKS" ]]; then
+    docs+=("tasks.md")
+fi
+
+# Output results
+if $JSON_MODE; then
+    # Build JSON array of documents
+    if [[ ${#docs[@]} -eq 0 ]]; then
+        json_docs="[]"
+    else
+        local escaped_docs=()
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)